config/initializers/simple_form.rb

```diff
@@ -0,0 +1,36 @@
+SimpleForm.setup do |config|
+  config.wrappers :default, class: :input, hint_class: :field_with_hint,
+                            error_class: :field_with_errors do |b|
+
+    b.use :html5
+
+    b.use :placeholder
+
+    b.optional :maxlength
+
+    b.optional :pattern
+
+    b.optional :min_max
+
+    b.optional :readonly
+
+    b.use :label_input
+    b.use :hint,  wrap_with: { tag: :span, class: :hint }
+    b.use :error, wrap_with: { tag: :span, class: :error }
+
+  end
+
+  config.default_wrapper = :default
+
+  config.boolean_style = :nested
+
+  config.button_class = 'btn'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.